packages/oui-select/src/select.controller.js

```diff
@@ -29,6 +32,14 @@ export default class {
             matchElement.html("{{$select.selected}}");
         }
 
+        if (isArray(this.disabledItems)) {
+            this.items = angular.copy(this.items).map(item =>
```

Comment:
We should use another property like `this.sortedItems` to keep the value of `this.item`.
What will happened when `items` or `disabled-items` is updated after the init ?